lib/src/types.dart

```diff
@@ -29,6 +29,13 @@ enum Reliability {
   lossy,
 }
 
+enum TrackSource {
+  unknown,
+  camera,
+  microphone,
+  screenShare,
```

Comment:
FYI: I also added a ScreenShareAudio type.. for when the screen share is shared with an audio track.